### ISSUE TITLE
Use editor theme color for editor splash screen

### ISSUE THREAD

**Issue by fire-forge** [godotengine/godot-proposals#4434]
### Describe the project you are working on

An app for making wallpapers

### Describe the problem or limitation you are having in your project

The editor splash screen always has a dark gray background color.

![image](https://user-images.githubusercontent.com/67974470/164946898-31537bcc-4b6d-4c32-9196-74cc45f2c7c7.png)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Use the theme editor settings to determine colors for the editor splash screen text and background color.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

The splash screen background color will be determined by the `interface/theme/base_color` editor setting. Text color would be determined by `interface/theme/icon_and_font_color`.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

The editor splash screen cannot be changed by a script.

### Is there a reason why this should be core and not an add-on in the asset library?

The editor splash screen cannot be changed by an add-on.

**Comment by KoBeWi** [comment]
This is actually the default clear color defined in the project settings.

**Comment by Calinou** [comment]
I agree with this, but last time I checked, you'd need to read the editor theme *before* the splash screen is initialized.
